[PID/(Q)EAA Issuance - Detailed Flow] Discovery (#309)
italia/eidas-it-wallet-docs: docs/en/pid-eaa-issuance.rst

    PID/(Q)EAA Issuance - Detailed flow

> **Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 

Reviewer: ```suggestion
**Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
```

> **Steps 1-4 (Discovery):** The User selects the PID/(Q)EAA Provider, only among the ones present into the appropriate trust lists, using the Wallet Instance, the Metadata for the selected PID/(Q)EAA Provider are processed by the Wallet Instance as defined in 
> `Trust Model section`_ Trust Evaluation Mechanism, of this specification.

Reviewer: ```suggestion
**Steps 1-4 (Discovery):** The User, using the Wallet Instance, selects the PID/(Q)EAA Provider from those listed in the list of trustworthy entities. The Wallet Instance then processes the Metadata for the selected PID/(Q)EAA Provider as defined in the `Trust Model section`_ Trust Evaluation Mechanism of this specification.
```

> `Trust Evaluation Mechanism <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/trust.html#trust-evaluation-mechanism>`_ section of this specification.

Reviewer 2: what do you think about using a relative path?

Reviewer: I did it @grausof ^

> `Trust Evaluation Mechanism <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/trust.html#trust-evaluation-mechanism>`_ section of this specification.

Reviewer: ```suggestion
`Trust Model section`_ Trust Evaluation Mechanism, of this specification.
```

Reviewer: @grausof wdyt^
